README.md

```diff
@@ -1,16 +1,10 @@
 # Workflow run cleanup action
 
-This action cleans up previously running instances of a workflow
-on the same branch. This accomplishes the task of automatically
-cancelling CI runs on pushes to the same branch, which is a common
-feature in most CI systems but currently not possible with
-GitHub actions.
+This action cleans up previously running instances of a workflow on the same branch. This accomplishes the task of automatically cancelling CI runs on pushes to the same branch, which is a common feature in most CI systems but currently not possible with GitHub actions.
```

Comment:
I don't think anything changed here? Could you undo the change that puts all the text on one line?